src/matlab2tikz.m

```diff
@@ -6244,3 +6202,89 @@ function checkDeprecatedEnvironment(minimalVersions)
     end
 end
 % ==============================================================================
+
+% DEPRECATED ===================================================================
```

Comment:
What should happen to those functions? Shouldn't we simply delete them now? GIT LOG allows us to travel back in time, in case we need to look something up.